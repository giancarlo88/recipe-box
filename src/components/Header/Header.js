import React from 'react'
import styled from 'styled-components'


const Header = styled.header`
background-color: rgba(125, 30, 50, .5);
color: green;
`

export default () => (
  <Header>
    <h1 className="App-title">Welcome to React</h1>
  </Header>
)
