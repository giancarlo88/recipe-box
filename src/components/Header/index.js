import React from 'react'
import styled from 'styled-components'

const StyledHeader = styled.header`
  align-items: center;
  background-color: rgba(125, 30, 50, .5);
  display: flex;
  height: 75px;
  justify-content: center;
  text-align: center;
`

const StyledHeaderTitle = styled.h1`
  font-family: 'Montserrat';
  font-size: 36px;
`

const Header = ({ text }) => (
  <StyledHeader>
    <StyledHeaderTitle>{text}</StyledHeaderTitle>
  </StyledHeader>
)

export default Header
