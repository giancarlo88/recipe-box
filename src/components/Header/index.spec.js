import React from 'react'
import Header from './index'
import { create } from 'react-test-renderer'
import { mount } from 'enzyme'

describe('Header', () => {
  const text = 'Header'

  it('should match a snapshot', () => {
    const props = {
      text
    }
    const tree = create(<Header {...props} />)
    expect(tree).toMatchSnapshot()
  })

  it('should render the text prop passed to it', () => {
    const component = mount(<Header text={text} />)
    expect(component.find('h1').text()).toBe(text)
  })
})
