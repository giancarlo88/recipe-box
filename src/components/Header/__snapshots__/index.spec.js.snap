// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match a snapshot 1`] = `
<header
  className="sc-bdVaJa oMxSe"
>
  <h1
    className="sc-bwzfXH kPEUNq"
  >
    Header
  </h1>
</header>
`;
